core/types/block.go

```diff
@@ -378,3 +378,43 @@ func (b *Block) Hash() common.Hash {
 }
 
 type Blocks []*Block
+
+// HeaderParentHashFromRLP returns the parentHash of an rlp-encoded
+// header-blob.
+// Fix-size gields in an RLP-encoded types.Header are:
+//
+// | name        | size   | type           |
+// | ------------|--------| ---------------|
+// | ParentHash  |   32   | common.Hash    |
+// | UncleHash   |   32   | common.Hash    |
+// | Coinbase    |   20   | common.Address |
+// | Root        |   32   | common.Hash    |
+// | TxHash      |   32   | common.Hash    |
+// | ReceiptHash |   32   | common.Hash    |
+// | Bloom       |   256  | Bloom          |
+// | GasLimit    |   8    |  uint64        |
+// | GasUsed     |   8    | uint64         |
+// | Time        |   8    | uint64         |
+// | MixDigest   |   32   | common.Hash    |
+// | Nonce       |   8    | BlockNonce     |
+//
+// Size of constant fields: `500`.
+//
+// | name        | size   | type           |  max
+// | ------------|--------| ---------------|-----------------------|
+// | Difficulty  | dynamic| *big.Int       | 0x5ad3c2c71bbff854908 (current mainnet TD: 76 bits) => 10 byte |
+// | Number      | dynamic| *big.Int       | 32 bits = 4 bytes              |
+// | Extra       | dynamic| []byte         | 32 byte + 65 byte = 97 byte (clique) |
```

Comment:
+20 bytes x signers in the middle for Clique

Reply:
Fixed